client/sites-dashboard/components/sites-site-item-thumbnail.tsx

```diff
@@ -36,6 +43,15 @@ export const SiteItemThumbnail = ( {
 }: SiteItemThumbnailProps ) => {
 	const { __ } = useI18n();
 	const classes = classNames( props.className, disallowSelection );
+	const newSiteId = site.is_creating ? site.ID : null;
+	const transferStatus = useCheckSiteTransferStatus( newSiteId );
+
+	const isTransferCompleted = transferStatus === transferStates.COMPLETE;
+	const isTransferring = transferStatus !== null && ! isTransferCompleted;
```

Comment:
by backend you mean extracting to it's own custom hook that's re-useable? If you php I don't think there's any need for that, there's an endpoint that already gives the transfer status and redux actions setup to fetch the status.

Reply:
>If you php I don't think there's any need for that, there's an endpoint that already gives the transfer status and redux actions setup to fetch the status.

OK 👍 

>by backend you mean extracting to it's own custom hook that's re-useable?

I meant to create an abstraction. I wasn't sure about the implementation, that's why I proposed moving to the BE. But it could well be in the FE as well.